.changes/v2.10.0/550-features.md

```diff
@@ -0,0 +1 @@
+* Add support for NSX-T Org Vdc [GH-550]
```

Comment:
I think it's better to use capital "VDC" in human-facing messages.
```suggestion
* Add support for NSX-T Org VDC [GH-550]
```

Reply:
This was fixed as well. Comment was missing.